src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -57,6 +57,7 @@
 
         <service id="form.choice_list_factory.cached" class="Symfony\Component\Form\ChoiceList\Factory\CachingFactoryDecorator">
             <argument type="service" id="form.choice_list_factory.property_access"/>
+            <tag name="kernel.reset" method="reset" />
```

Comment:
To preserve cross deps, the tag should be removed in FrameworkExtension when the method doesn't exist.

Reply:
Not sure we have to care about that, there is a conflict https://github.com/symfony/symfony/blob/4.2/src/Symfony/Bundle/FrameworkBundle/composer.json#L71.

Reply:
The conflict won't prevent installing 4.2.0, where the method doesnt exist.

Reply:
Understood, done! Thanks